### ISSUE TITLE
grid not aligned with origin

### ISSUE THREAD

**Issue by jeff-dh** [jreiberkyle/jupyterscad#11]
![grid_not_aligned_with_origin](https://github.com/jreiberkyle/jupyterscad/assets/1105041/0ad8d5a0-2b16-4aea-8df4-a703baa32c38)

**Comment by jreiberkyle** [comment]
Ohh interesting. I'll bet this is due to the logic for determining the grid min/max. Thank you for finding this!

**Comment by jeff-dh** [comment]
got a solution for this.... trying the auto grid stuff and if successfull gonna create a pr for both together....

**Comment by jreiberkyle** [comment]
Awesome! My preference would be two prs for the two different issues. Easier and faster to review. Thank you!

**Comment by jreiberkyle** [comment]
A little context: I want to get to v1 release asap, which includes bug fixes but doesn’t necessarily include autoscaling the grid. More features means potentially more bugs. So, likely the grid origin fix will go into v1 (I hope to release by oct 30) and autoscaling will go into v1.1.

**Comment by jeff-dh** [comment]
hmmm autoscaling is a single line...... I don't think there's too much potential for bugs...

```python
       if unit == GRID_AUTO_UNIT:
            unit = 10**math.floor(math.log10(max_extend))
```
that's pretty much it, but it's up to you....

**Comment by jreiberkyle** [comment]
Closed by #15
